src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
+        rdfs:label "gas turbine process"
+    
+    SubClassOf: 
+        OEO_00020003,
+        OEO_00000532 some OEO_00000099,
+        OEO_00010234 some OEO_00000007,
+        OEO_00010235 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00140038,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00000185
+    
+    
+Class: OEO_00310028
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A steam power unit is a power generating unit using steam.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "steam power unit"
+    
+    SubClassOf: 
+        OEO_00000334,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000396
+    
+    
+Class: OEO_00310029
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A combined cycle power plant (CCGT) is a gas power plant that has gas fired power units, steam power units, gas power units and heat recovery steam units as parts.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CCGT",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "combined cycle power plant"
+    
+    SubClassOf: 
+        OEO_00000184,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00310028,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00310030,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00310031
+    
+    
+Class: OEO_00310030
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas power unit is a power generating unit using gas.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "gas power unit"
+    
+    SubClassOf: 
+        OEO_00000334,
+        OEO_00000503 some OEO_00000099,
+        OEO_00010234 some OEO_00000007,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000185
+    
+    
+Class: OEO_00310031
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat recovering steam unit is an energy transformation unit that recovers heat from steam.",
```

Comment:
See above.  If it is a unit that has a "heat recovery steam generator", then it will recover heat for generating steam.
If it where not, hot steam goes in … and what comes out?  How is energy being recovered?  Makes no sense engineering-wise.

Reply:
I propose a new definition:
`A heat recovering steam unit is an energy transformation unit that uses recovered heat to generate steam.`
Then, it will also be not transferred later as a subclass of `heat generating unit` (see discussion above) but stay as an `energy transformation unit`.